code/utilities.py

```diff
@@ -6,15 +6,16 @@
 from logging.handlers import RotatingFileHandler
 
 ## Config
-CONFIG_OPTIONS = {}         # This'll be populated on import
-DEBUG_LEVEL_KEY = "debug_level"
-CONFIG_NAME = "config.json"	# The name of the config file
-DIRS_FROM_ROOT = 1			# How many directories away this script is from the root
+CONFIG_OPTIONS = {}                     # This'll be populated on import
+CONFIG_NAME = "config.json"	            # The name of the config file
+DEV_CONFIG_NAME = "config.dev.json"     # The name of the dev config file (overrides properties stored in the normal and prod config files)
+PROD_CONFIG_NAME = "config.prod.json"   # The name of the prod config file (overrides properties stored in the normal config file)
+DIRS_FROM_ROOT = 1			            # How many directories away this script is from the root
 PLATFORM = sys.platform
 
 
 def get_root_path():
-	## -1 includes this script itself in the realpath
```

Comment:
Is there a nice way to covert tabs to spaces automagically, like some kind of Github CI pipeline or something?